resources/services/ecs/task_definitions.go

```diff
@@ -606,17 +633,12 @@ func fetchEcsTaskDefinitions(ctx context.Context, meta schema.ClientMeta, parent
 		if len(listClustersOutput.TaskDefinitionArns) == 0 {
 			return nil
 		}
+
 		for _, t := range listClustersOutput.TaskDefinitionArns {
-			describeClusterOutput, err := svc.DescribeTaskDefinition(ctx, &ecs.DescribeTaskDefinitionInput{TaskDefinition: &t}, func(o *ecs.Options) {
-				o.Region = region
-			})
+			err := fetchEcsTaskDefinition(ctx, res, svc, region, t)
```

Comment:
Would maybe make this call in parallel

Reply:
I agree that would be an improvement, I was trying trying to get this out as quickly as possible as this would reduce fetch times by 50%